ads/_config.js

```diff
@@ -837,6 +837,8 @@ const adConfig = jsonConfiguration({
 
   'postquare': {},
 
+  'ppstudio': {},
```

Comment:
I see that you haven't implemented `renderStart`, if you do we can actually render the ad a bit faster when we know the ad is ready.

Reply:
Dear @calebcordry 
I add render `global.context.renderStart();` after `loadScript` parameter `opt_cb`.

PPStudio is an ad creative creation tool for google ad manager (GAM) or other 3-party delivery system. 

When the script load it will not NO AD condition.

Hope it complies with your requirements.